addons/im_livechat/static/tests/embed/composer_patch_tests.js

```diff
@@ -0,0 +1,29 @@
+/* @odoo-module */
+
+import { startServer } from "@bus/../tests/helpers/mock_python_environment";
+
+import { loadDefaultConfig, start } from "@im_livechat/../tests/embed/helper/test_utils";
+
+import { triggerHotkey } from "@web/../tests/helpers/utils";
+import { click, contains, insertText } from "@web/../tests/utils";
+
+QUnit.module("composer patch");
+
+QUnit.test("File upload only available when authorized", async () => {
+    const pyEnv = await startServer();
+    await loadDefaultConfig();
+    const env = await start();
+    await click(".o-livechat-LivechatButton");
+    await contains(".o-mail-ChatWindow");
+    await insertText(".o-mail-Composer-input", "Hello, please enable file uploading!");
+    triggerHotkey("Enter");
+    await contains(".o-mail-Message-content", { text: "Hello, please enable file uploading!" });
```

Comment:
These 3 LOCs are unnecessary for the good working of the test, but whatever 😅

Reply:
Well, without any message the thread is not created on the server so we cannot toggle the attachment upload! I removed the last line since this assertion is useless, but the first two are required and reflect what really happens.